why-polars-for-data-cleaning/index.qmd

```diff
@@ -0,0 +1,202 @@
+---
+title: "Why use Polars for data cleaning?"
+description: "The key to useful and meaningful data is data that is good quality and tidy. Getting data to be tidy and higher quality is an often undervalued and underestimated activity, at the same time being very labour intensive. But it is essential to have higher quality and tidier data to have better quality results from the analysis. This post explains why we chose to use Polars for data cleaning and processing tasks."
```

Comment:
```suggestion
title: "Why Polars"
description: "The key to useful and meaningful data is data that is good quality and tidy. Getting data to be tidy and higher quality is an often undervalued and underestimated activity, at the same time being very labour intensive. But it is essential to have higher quality and tidier data to have better quality results from the analysis. This post explains why we chose to use Polars for data processing tasks."
```

I realized from our meeting and discussion that there is some miscommunication and misunderstanding about the tool, and I think it stems from the word "cleaning". So instead, I'm changing everything to reference "processing" rather than "clean" and hopefully that will help align understandings :relaxed: